cloud: support always-on dev tier (#9587)
pingcap/docs: tidb-cloud/release-notes-2022.md

This page lists the release notes of [TiDB Cloud](https://en.pingcap.com/tidb-cloud/) in 2022.

## July xx, 2022

* Behavior changes to Developer Tier

>     * The Developer Tier cluster will not be deleted after 7 days of inactivity so you can still use it at any time until the one-year free trial ends. After a few days of inactivity, the Developer Tier cluster will be hibernated automatically. To resume the cluster, either send a new connection to the cluster or click the **Resume** button on the TiDB Cloud console.

Reviewer: This does not need to be stated in the release note

This page lists the release notes of [TiDB Cloud](https://en.pingcap.com/tidb-cloud/) in 2022.

## July xx, 2022

> * Behavior changes to new Developer Tier clusters

Reviewer 2: ```suggestion
* Supports TiDB Cloud API. TiDB Cloud API is a RESTful based interface which can interact with TiDB Cloud Service. Through this API, customers can manage resources such as clusters automatically and efficiently. For more information, refer to [API Reference](https://docs.pingcap.com/tidbcloud/api/v1).
* Behavior changes to new Developer Tier clusters
```

This page lists the release notes of [TiDB Cloud](https://en.pingcap.com/tidb-cloud/) in 2022.

## July xx, 2022

> * Supports TiDB Cloud API. TiDB Cloud API is a RESTful based interface, which can interact with TiDB Cloud services. Through this API, you can manage resources such as clusters automatically and efficiently. For more information, refer to [API Reference](https://docs.pingcap.com/tidbcloud/api/v1).

Reviewer 3: "You can manage something automatically" looks a bit strange.
```suggestion
* Supports TiDB Cloud API. TiDB Cloud API is a RESTful-based interface, which can interact with TiDB Cloud services. Through this API, you can manage resources such as clusters automatically and efficiently. For more information, refer to [API Reference](https://docs.pingcap.com/tidbcloud/api/v1).
```

> * Supports TiDB Cloud API. TiDB Cloud API is a RESTful based interface which can interact with TiDB Cloud Service. Through this API, customers can manage resources such as clusters automatically and efficiently. For more information, refer to [API Reference](https://docs.pingcap.com/tidbcloud/api/v1).

Author: ```suggestion
* Supports TiDB Cloud API. TiDB Cloud API is a RESTful based interface, which can interact with TiDB Cloud services. Through this API, you can manage resources such as clusters automatically and efficiently. For more information, refer to [API Reference](https://docs.pingcap.com/tidbcloud/api/v1).
```

Reviewer 2: LGTM

> * //TODO: Add the **API Key** tab in the TiDB Cloud console.

Reviewer 2: ```suggestion
* Add the **API Key** tab in the **Orgnization Settings** page. You can create and manage API keys required for using API.
```

Reviewer 4: > 

```suggestion
* Add the **API Key** tab in the **Organization Settings** page. You can create and manage API keys required for using TiDB Cloud API.
```

> * //TODO: Add the **API Key** tab in the TiDB Cloud console.

Reviewer 5: ```suggestion
* //TODO: Add the **API Keys** tab in the TiDB Cloud console.
```

> * Add the **API Key** tab in the **Organization Settings** page. You can create and manage API keys required for using TiDB Cloud API.

Reviewer 5: ```suggestion
* Add the **API Keys** tab in the **Organization Settings** page. You can create and manage API keys required for using TiDB Cloud API.
```

> * Behavior changes to new Developer Tier clusters

Reviewer 3: ```suggestion
* Behavior changes for new Developer Tier clusters
```

This page lists the release notes of [TiDB Cloud](https://en.pingcap.com/tidb-cloud/) in 2022.

> ## July 26, 2022

Reviewer: LGTM

* Support TiDB Cloud API.

>     TiDB Cloud API is a RESTful-based interface, which can interact with TiDB Cloud services. Through this API, you can manage resources such as clusters automatically and efficiently.

Reviewer 3: ```suggestion
    TiDB Cloud API is a RESTful-based interface, which can interact with TiDB Cloud services. Through this API, you can manage resources such as clusters efficiently.
```

>     In the TiDB Cloud console, you can find the **API Key** tab on the **Organization Settings** page. With this tab, you can create and manage API keys required for using TiDB Cloud API.

Reviewer 2: ```suggestion
    In the TiDB Cloud console, you can find the **API Keys** tab on the **Organization Settings** page. With this tab, you can create and manage API keys required for using TiDB Cloud API.
```
